packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -317,6 +318,79 @@ curl -v -X POST \
 ##### Response Example
 
 
+```json
+{
+  "id": "00ub0oNGTSWTBKOLGLNR",
+  "status": "ACTIVE",
+  "created": "2013-07-02T21:36:25.344Z",
+  "activated": null,
+  "statusChanged": null,
+  "lastLogin": null,
+  "lastUpdated": "2013-07-02T21:36:25.344Z",
+  "passwordChanged": "2013-07-02T21:36:25.344Z",
+  "profile": {
+    "firstName": "Isaac",
+    "lastName": "Brock",
+    "email": "isaac.brock@example.com",
+    "login": "isaac.brock@example.com",
+    "mobilePhone": "555-415-1337"
+  },
+  "credentials": {
+    "password": {},
+    "provider": {
+      "type": "IMPORT",
+      "name": "IMPORT"
+    }
+  },
+  "_links": {
+    "activate": {
+      "href": "https://${yourOktaDomain}/api/v1/users/00ub0oNGTSWTBKOLGLNR/lifecycle/activate"
+    },
+    "self": {
+      "href": "https://${yourOktaDomain}/api/v1/users/00ub0oNGTSWTBKOLGLNR"
+    }
+  }
+}
+```
+
+#### Create User with Password Import Inline Hook
+
+
+Creates a user with a specified [password hook](#password-hook-object). See [Password Import Inline Hook](/docs/reference/password-hook/) for more details.
+The new user is able to login immediately after activation with the specified password.
+This flow is common when migrating users from another data store in cases where we wish to allow the users to retain their current passwords.
+
+> Important: Do not generate or send a one-time activation token when activating users with an password hook.  Users should login with their existing password to be imported using the password import inline hook.
+
```

Comment:
Maybe helpful to also explicitly say here that `POST /users` must be called with query param `activate=false` or the call will not work.

Reply:
Just wanted to keep it consistent with the existing doc for `Password Imporing using Hash`, and so did not want to add any extra details.